### PR TITLE
Replace rivets send page with react send page

### PR DESCRIPTION
**Pull request by booshibooshi**

### CODE REVIEW

File: tool/site/client-react/src/pages/sendPage.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  }
+
+  return <>
+    <Label isPageHeading>Error</Label>
+    <ErrorMessage>An unknown error occurred.</ErrorMessage>
+  </>
+}
+
+interface SendResult {
+  prescription_ids: string[]
```
Comment (mfjarvis) [inline tool/site/client-react/src/pages/sendPage.tsx:97]: Will this page fail to render for a bulk send? I can't find a valid test pack to check

Comment (mfjarvis) [reply tool/site/client-react/src/pages/sendPage.tsx:97]: Read a bit further and answered my own question - it'll just render the first one. Think that's fine for now

Comment (booshibooshi) [reply tool/site/client-react/src/pages/sendPage.tsx:97]: Yeah bulk send not supported for the moment, put in a placeholder, but will probably change when we decide how to handle pagination in new client

**Review by mfjarvis — APPROVED** [review]

File: tool/site/server_hapi/src/routes/prescribe/send.ts [hunk 2]
```diff
@@ -45,15 +50,17 @@ export default [
...
           prescription_ids: prescriptionIds,
           prescription_id: prescriptionIds[0],
           success: true,
           request_xml: sendRequestHl7,
           request: sendRequest,
           response: sendResponse.fhirResponse,
           response_xml: sendResponse.spineResponse
-        }).code(200)
+        }
+        setSessionValue(`signature_token_${signatureToken}`, sendResult, request)
```
Comment (mfjarvis) [inline tool/site/server_hapi/src/routes/prescribe/send.ts:62]: This key seems a bit odd (we're not storing a signature token here). Session storage as a whole needs some thought, so not overly worried about changing it right now

**Pull request merged**